x-pack/plugins/uptime/public/components/monitor/monitor_title.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useMonitorId } from '../../hooks';
+import { monitorStatusSelector } from '../../state/selectors';
+import { EnableMonitorAlert } from '../overview/monitor_list/columns/enable_alert';
+import { Ping } from '../../../common/runtime_types/ping';
+
+const isAutogeneratedId = (id: string) => {
+  const autoGeneratedId = /^auto-(icmp|http|tcp)-OX[A-F0-9]{16}-[a-f0-9]{16}/;
+  return autoGeneratedId.test(id);
+};
+
+// For monitors with no explicit ID, we display the URL instead of the
```

Comment:
Thanks for the comment!

Reply:
I don't think this case is working right, here's what I see:

![image](https://user-images.githubusercontent.com/131427/102235906-7901d580-3eb8-11eb-8a34-958ef51596cf.png)

Reply:
i fixed this